src/app/components/splitbutton/splitbutton.ts

```diff
@@ -32,7 +32,7 @@ type SplitButtonIconPosition = 'left' | 'right';
                     (click)="onDefaultButtonClick($event)"
                     [disabled]="disabled"
                     [attr.tabindex]="tabindex"
-                    [attr.aria-label]="buttonProps?.['aria-label'] || label"
+                    [ariaLabel]="buttonProps?.['ariaLabel'] || label"
```

Comment:
I think the correct way is `attr.aria-label` in Angular. Are you sure about this change?
Maybe @cetincakiroglu can confirm it

Reply:
@SoyDiego Since we are using pbutton as a directive and pbutton has the property named "ariaLabel", we can pass the property name as "ariaLabel".

Reply:
Thanks for clarify 😊

Reply:
> Thanks for clarify 😊

No problem 🙂